packages/dialtone-vue2/components/hovercard/hovercard.mdx

```diff
@@ -0,0 +1,51 @@
+import { Canvas, Story, Subtitle, Controls, Meta } from '@storybook/blocks';
+
+import * as HovercardStories from './hovercard.stories';
+
+<Meta of={HovercardStories}/>
+
+# Hovercard
+
+<Subtitle>
+  The Hovercard component provides a popover-like experience for displaying additional information.
+</Subtitle>
+
+## Base Style
+
+The hovercard will appear upon the mouse entering the anchor, with a delay of 300 milliseconds.
+It will remain open as long as the mouse cursor is over either the open card or the anchor.
+
+<Canvas>
+  <Story of={HovercardStories.Default} />
+</Canvas>
+
+## When there are many hovercards
+
+Initially, there is a 300ms delay when numerous hovercards are present.
+However, when transitioning from one anchor to another, the delay is determined by the mouse movement time.
+This implies that the closer the anchors are, the faster the hovercard appears.
+Move the mouse over the avatars to reveal the hovercards in the example.
```

Comment:
Should we explain here that is dependant of a global timer that need to be set it up?

Reply:
Yes, will do

Reply:
Fixed in latest changes